sdk/packages/modal/src/components/nightly-selector/nightly-selector.ts

```diff
@@ -127,6 +127,15 @@ export class NightlySelector extends LitElement {
     this.onSelectWallet(this.currentWalletName)
   }
 
+  downloadApp = () => {
+    const a = document.createElement('a')
```

Comment:
no, don't do it this way: use window.open(<url>, '_blank'), this thing here is overengineered

Reply:
fix